common/workunit/workunit.hpp

```diff
@@ -527,6 +527,7 @@ interface IConstWUClusterInfo : extends IInterface
     virtual unsigned getRoxieRedundancy() const = 0;
     virtual unsigned getChannelsPerNode() const = 0;
     virtual int getRoxieReplicateOffset() const = 0;
+    virtual IStringVal & getAlias(IStringVal & str) const = 0;
```

Comment:
I want to phase out use of IStringVal... any reason this can't be a const char * queryAlias() method?